Change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.    plt.show()
</pre><ol><li>Here if you observe,the 3rd plot, then yticks is different from the original figure in book1.</li></ol><p>This happens because many a times code in github is not updated as current figure. In this case you need to make some changes, I suggest to read text for this figure, first understand what actually figure is about then may be change some code to make figure same as given in book.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>